docs/product/issues/issue-details/replay-issues/hydration-error.mdx

```diff
@@ -0,0 +1,48 @@
+---
+
+Hydration errors are a React-specific problem that happen when the initial UI does not match what was rendered on the server. They can result in extra work for the browser, and a slower pageload experience for users.
+
+## Prerequisites for Seeing Hydration Error Diff Tools
+
+Hydration Errors can come into your issue stream either as regular error events, or they could be detected inside a replay session and converted into an issue by the server. Read more about
+<PlatformLink to="/session-replay/issue-types/">configuring hydration errors</PlatformLink> to learn about the differences.
+
+If the hydration error event instance has an associated replay session, then you will see diff tools on the issue details page to make debugging the hydration error easier.
+
+## Debugging Hydration Errors
+
+Hydration errors represent a mismatch between the server-rendered HTML, and the client-rendered HTML. So it's beneficial to compare the before and after state of a webpage to understand what's changed, and decide how to make those two states consistent. To do this, we provide a few diff tools to see the differences -- both visually and by looking at the rendered HTML itself.
+
+Generally the visual tools are a good place to start to identify the issue, but keep in mind that sometimes differences can appear below the fold, or there can be differences in hidden DOM nodes or attributes, so inspecting the HTML is important too.
+
+### Example Error
+
+In the example below we'll look at a hydration error and see how each tool helps to debug the problem.
+
+The problem is that code highlighting is not applied on the server. When the server renders the PHP code block in the bottom-center of the page, all the code has a white font color. After hydration syntax highlighting is applied, the code snippet is colorful.
+
+#### Image Slider Tool
+
+The image slider tool allows for comparing the two page states by overlaying the images on top of each other. Click and drag the purple line to reveal one image or the other.
+
+![image slider tool](./img/hydration-error-image-slider-tool.png)
+
+#### Side by Side Image Tool
+
+View images side by side to see large chunks of the page that may differ.
+
+![side by side image tool](./img/hydration-error-side-by-side-image-tool.png)
+
+#### HTML Diff Tool
+
+The HTML diff tool is a basic HTML comparison. It is useful if the hydration error is caused by hidden DOM nodes, different attributes on a DOM node, or mismatched content below the fold.
+
+![html diff tool](./img/hydration-error-html-diff-tool.png)
+
+## Privacy
+
+The diff tools all rely on replay data, so any PII data within the webpage will be masked or blocked before being sent to the server. Read more about [Protecting user privacy in Session Replay](https://docs.sentry.io/security-legal-pii/scrubbing/protecting-user-privacy/) and <PlatformLink to="/session-replay/privacy/">how to configure Session Replay to maintain user privacy</PlatformLink>.
```

Comment:
```suggestion
The diff tools all rely on replay data, so any PII data within the webpage will be masked or blocked before being sent to the server. Read more about [Protecting user privacy in Session Replay](https://docs.sentry.io/security-legal-pii/scrubbing/protecting-user-privacy/) and [how to configure Session Replay to maintain user privacy](https://docs.sentry.io/platforms/javascript/session-replay/privacy/).
```

Again, replacing with a link so that they don't have to click JS when it's the only option